src/frontend/js/components/DashboardItem/DashboardItemOrder.tsx

```diff
@@ -0,0 +1,72 @@
+import { defineMessages, useIntl } from 'react-intl';
+import { Enrollment, Order } from '../../types/Joanie';
+import { DashboardItem, DEMO_IMAGE_URL } from './index';
+import { Icon } from '../Icon';
+import { Button } from '../Button';
+import { DashboardSubItemsList } from './DashboardSubItemsList';
+import { DashboardSubItem } from './DashboardSubItem';
+import { DashboardItemEnrollmentFooter } from './DashboardItemEnrollment';
+
+const messages = defineMessages({
+  accessCourse: {
+    id: 'components.DashboardItemEnrollment.gotoCourse',
+    description: '',
+    defaultMessage: 'ACCESS COURSE',
+  },
+});
+
+interface DashboardItemOrderProps {
+  order: Order;
+}
+
+export const DashboardItemOrder = ({ order }: DashboardItemOrderProps) => {
+  const course = order.courseEntity;
+  if (!course) {
+    throw new Error('Order must provide course attribute');
+  }
+
+  const intl = useIntl();
+  return (
+    <DashboardItem
+      title={course.title}
+      headRef={'Ref. ' + course.code}
+      imageUrl={DEMO_IMAGE_URL}
+      footer={
+        <>
+          <div className="dashboard-course-run__block__status">
+            <Icon name="icon-school" />
+            <div>
+              For PR @jbpenrath: Don't know which data to use here, all course course_runs ? But
+              does it makes sense for a gradeo product ?
+            </div>
+          </div>
+          <Button
+            label={"@jbpenrath: How to get the course's url ?"}
```

Comment:
There will be a dashboard course detail view, I think with following url 
`/dashboard/courses/:course_code`